src/test/java/uk/gov/companieshouse/controller/PaymentControllerTest.java

```diff
@@ -37,6 +38,14 @@
     private static final String PAYMENT_URI = "/dissolution-request/{company-number}/payment";
```

Comment:
could we add some coverage to `POST /dissolution-request/submit` now that it is in master?